### PR TITLE
First draft of deployment documentation

### PR DESCRIPTION
**Pull request by SubJunk**

### CODE REVIEW

**Comment by SubJunk** [comment]
@UniversalMediaServer/developers I documented the steps I took during the last deployment and put them here. I also will need to add the files I referenced in the document before it's helpful, and then there's a whole other discussion to be had about permissions. But it's a start :)

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+1.  Be on latest `master` branch
+1.  Pull translations from Crowdin and commit them
+1.  Update changelog with the changes since the last version, and the release date
+1.  Decide which kind of version bump is necessary. Bugfixes or refactoring should bump the third number, others should bump the second one.
+1.  Update the versions in the changelog and pom.xml (lines 33 and 66) including removing "-SNAPSHOT" from line 33
+1.  Create a tag of the new version (e.g. 6.7.2, not v6.7.2)
+1.  Push your commits and new tag
+
+## Build:
+1.  Run build.bat on Windows, which generates releases for Windows and Linux
```
Comment (Nadahar) [inline DEPLOYMENT.md]: @SubJunk Where is ```build.bat``` located?

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+1.  Append version numbers to the files
```
Comment (Nadahar) [inline DEPLOYMENT.md]: @SubJunk This should be exlained furter, what's the rules?

File: DEPLOYMENT.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+1.  Switch to macOS
+1.  Pull latest master
+1.  Run build.sh
```
Comment (Nadahar) [inline DEPLOYMENT.md]: @SubJunk Where is ```build.sh``` located?

File: DEPLOYMENT.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+1.  Append version number to the generated .dmg file
```
Comment (Nadahar) [inline DEPLOYMENT.md]: @SubJunk Again, exact rules for naming.

File: DEPLOYMENT.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+## Upload:
+1.  Upload the 3 release files to spirton.com
+1.  Upload to FossHub by pointing the WGET tool to the file URLs
+1.  Add the release metadata to FossHub (platform, version)
+1.  Toggle visibility of older release files
```
Comment (Nadahar) [inline DEPLOYMENT.md]: @SubJunk Is this on FossHub or spirton.com?

File: DEPLOYMENT.md (hunk 1 above)
Comment (SubJunk) [inline DEPLOYMENT.md]: Like I said in the comment https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1335#issuecomment-313900973 I need to add those

File: DEPLOYMENT.md (hunk 2 above)
Comment (SubJunk) [inline DEPLOYMENT.md]: Those are explained on line 9

File: DEPLOYMENT.md (hunk 4 above)
Comment (SubJunk) [inline DEPLOYMENT.md]: Line 9

File: DEPLOYMENT.md (hunk 3 above)
Comment (SubJunk) [inline DEPLOYMENT.md]: https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1335#issuecomment-313900973

File: DEPLOYMENT.md (hunk 5 above)
Comment (SubJunk) [inline DEPLOYMENT.md]: FossHub. I can make that more clear

File: DEPLOYMENT.md (hunk 1 above)
Comment (Nadahar) [inline DEPLOYMENT.md]: I didn't see that, sorry. Creating a "Release" folder in the repository and putting this file and the "help files" there would probably be my take.

File: DEPLOYMENT.md (hunk 2 above)
Comment (Nadahar) [inline DEPLOYMENT.md]: I wasn't thinking of the version number itself, but how to modify the filename - the file name pattern if you will.

File: DEPLOYMENT.md (hunk 3 above)
Comment (Nadahar) [inline DEPLOYMENT.md]: https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1335/files#r126417379

File: DEPLOYMENT.md (hunk 4 above)
Comment (Nadahar) [inline DEPLOYMENT.md]: https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1335/files#r126417725

**Pull request closed without merging**